livedata/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -465,6 +465,8 @@ internal class ChannelControllerImpl(
         }
 
         newMessage.user = domainImpl.currentUser
+        // TODO: type should be a sealed/class or enum at the client level
+        newMessage.type = if (newMessage.text.startsWith("/")) { "ephemeral" } else { "regular" }
```

Comment:
Is it safe to assume that message is ephemeral or not by `tartsWith("/")` condition? 
E.g. What if the backend decides that message with text like `"/////"` is of `regular` type instead? Isn't it going to break something?

Reply:
no it will just update once the backend tells us what it really is. it's just a sensible default so that we actually show the message in the channel list (it filters on ==regular)

Reply:
So the filter will not allow showing messages starting with "/", right? I think it might cause some confusion in the future. Anyway, it's rare to start a chat message with this char 😄

Reply:
so if you send a message that is something like `/hi there`
it won't show up in the channel list while offline. as soon as the MessageEvent is received it will update to the correct state. (or actually probably raise an error that /hi is not defined)